parser_library/test/context/context_test.cpp

```diff
@@ -65,21 +65,43 @@ TEST(context_id_storage, case_insensitive)
 TEST(context, create_global_var)
 {
     hlasm_context ctx;
-
+    diagnostic_op_consumer_container diag;
 
     auto idx = ctx.ids().add("var");
 
-    auto glob = ctx.create_global_variable<int>(idx, true);
+    auto glob = ctx.create_global_variable<C_t>(idx, true, range(), diag);
```

Comment:
why does the test need the change in variable type from A_t to C_t?

Reply:
I just didn't want the two tests (`create_global_var` and `create_global_var_different_types`) to work with the same variable types. That's the only reason.